web/src/routes/(user)/people/+page.svelte

```diff
@@ -451,14 +451,31 @@
             on:merge-people={() => handleMergePeople(person)}
             on:hide-person={() => handleHidePerson(person)}
           />
-        {/if}
-      {/each}
+        {/each}
+      {:else}
+        {#each people as person, index (person.id)}
+          {#if !person.isHidden}
+            <PeopleCard
+              {person}
+              preload={index < 20}
+              on:change-name={() => handleChangeName(person)}
+              on:set-birth-date={() => handleSetBirthDate(person)}
+              on:merge-people={() => handleMergePeople(person)}
+              on:hide-person={() => handleHidePerson(person)}
+            />
+          {/if}
+        {/each}
+      {/if}
     </div>
   {:else}
     <div class="flex min-h-[calc(66vh_-_11rem)] w-full place-content-center items-center dark:text-white">
       <div class="flex flex-col content-center items-center text-center">
         <Icon path={mdiAccountOff} size="3.5em" />
-        <p class="mt-5 text-3xl font-medium">No people</p>
+        <p class="mt-5 text-3xl font-medium">
+          No people {#if searchName}
```

Comment:
I would prefer if the if starts in the next line

Reply:
Yeah I was worried about the space between `people` and `with`

Reply:
Oh then don't use an if but a ternary in the same line? Something like `No people{searchName ? ' with name ${searchName}' : ''}` (there should be backticks ofc since it's a template string)